### PR TITLE
Add papertrail

### PR DESCRIPTION
**Pull request by phirefly**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -20,6 +20,7 @@ gem 'jquery-turbolinks'
 gem 'newrelic_rpm'
 gem 'omniauth-myusa', git: 'https://github.com/18F/omniauth-myusa.git'
 gem 'paperclip'
+gem 'paper_trail', '~> 4.0.0.rc'
```
Comment (cmc333333) [inline Gemfile]: @afeld asked me before if the specific version number was needed, and if so, to add a comment explaining. Now, you too have this request.

**Comment by cmc333333** [comment]
Easy peasy!

File: Gemfile (hunk 1 above)
Comment (phirefly) [inline Gemfile]: I added the version per the gem's installation instructions, not for any other reason. I will go ahead and remove this.

**Comment by cmc333333** [comment]
:+1:

**Pull request merged**